diff --git a/cage.c b/cage.c
index e6c2cb1..9108526 100644
--- a/cage.c
+++ b/cage.c
@@ -296,15 +296,11 @@ main(int argc, char *argv[])
 int
 main(int argc, char *argv[])
 {
-	struct cg_server server = {.log_level = WLR_INFO};
+	struct cg_server server = {.log_level = WLR_ERROR};
 	struct wl_event_source *sigchld_source = NULL;
 	pid_t pid = 0;
 	int ret = 0, app_ret = 0;
 
-#ifdef DEBUG
-	server.log_level = WLR_DEBUG;
-#endif
-
 	if (!parse_args(&server, argc, argv)) {
 		return 1;
 	}
